### DISCUSSION TITLE
[400] PORTAL

### DISCUSSION THREAD

**Discussion by LW233333333** [pjy612/SteamManifestCache#33] [Share]
"402"
					{
						"DecryptionKey"		"1d9699c90f13a7549a97e5935b9eb152502aa664ce6944a56573c5f9a5c6d2f1"
					}
					"409"
					{
						"DecryptionKey"		"28cf5b133c63b6f5f1fab9d376b3ce35c7dd327505459288ae8d772a262c48f4"
					}
					"410"
					{
						"DecryptionKey"		"6bb84a4ff29b21f5f3e52a121c1293ad75c92ba8ad0b050fa691a9c15164a5e9"
					}
					"401"
					{
						"DecryptionKey"		"3ed55d26ffdf7f6c5e228ec60127ddc1712f341cfb082cadb9cebbd0f64d0d7d"
					}

这样可以吗？

**Comment by LW233333333** [comment]
code ：AA686192E9F2BD12BF989AB6DD226738DE8BAFDA

**Comment by pjy612** [comment]
传送门的话 应该没人没有吧。。。

**Reply by LW233333333** [reply]
总有人没有的吧，而且现在还没打折呢。。。

**Reply by pjy612** [reply]
主要是老游戏 隔壁仓库 已经很全了
